packages/app/src/components/swap/swap.tsx

```diff
@@ -107,18 +108,22 @@ const Swap: FC<Props> = ({
     }
   }, [calcOutGivenIn, receiveTokenAmount]);
 
+  // display progress during approval
+  useEffect(() => {
+    console.log("approveTx", approveTx.status);
+    setApprovalState(approveTx.status);
+  }, [approveTx, setApprovalState]);
```

Comment:
remove

Reply:
do you mean the console.log?

Reply:
No the entire thing this should not be in here.